Source/DafnyCore/DafnyOptions.cs

```diff
@@ -380,6 +380,8 @@ public enum IncludesModes {
     [CanBeNull] private TestGenerationOptions testGenOptions = null;
     public bool ExtractCounterexample = false;
 
+    public bool ShowProofObligationExpressions = false;
```

Comment:
Please don't add any fields to `DafnyOptions`, nor use `DafnyOptions.RegisterLegacyBinding`. The right way is to use `DafnyOptions.RegisterLegacyUI`.

Could you fix this is your next related PR?